docs/PacketFence_Administration_Guide.asciidoc

```diff
@@ -890,14 +889,7 @@ Option 1: Authentication against Active Directory (AD)
 
 CAUTION: If you are using an Active/Active or Active/Passive cluster, please follow the instructions under 'Option 1b' since the instructions below do not currently work in a cluster.
 
-In order to have domain authentication working properly, you need to enable IP forwarding on your server. To do it permanently, look in the `/etc/sysctl.conf`, and set the following line:
-
-  # Controls IP packet forwarding
-  net.ipv4.ip_forward = 1
-
-Now execute `sysctl -p` to apply the configuration
-
-Next, go in the Administration interface under *Configuration -> Domains*.
+Go in the Administration interface under *Configuration -> Domains*.
```

Comment:
That's not correct regarding the "new admin".

It would be in "Configuration -> Policies and Access Control -> Domains"